Ray.jl/test/runtests.jl

```diff
@@ -20,8 +17,8 @@ include("utils.jl")
     setup_ray_head_node() do
         include("function_manager.jl")
         setup_core_worker() do
-            include("task.jl")
             include("object_store.jl")
+            include("task.jl")
```

Comment:
why swap the order of these?

Reply:
Because `task.jl` now depends on `Ray.get`. Makes more sense to test `object_store.jl` first